skaffold.yaml

```diff
@@ -1,46 +1,118 @@
+# Top level variables
+.aea_version: &aea_version
+  AEA_VERSION: "1.16.0"
+.docker_args: &docker_args
+  dockerfile: Dockerfile
+.release_docker_args: &release_docker_args
+  <<: *docker_args
+  noCache: true
+
+# Agents to be built
+.oracle: &oracle
+  <<: *aea_version
+  AEA_AGENT: "valory/oracle:0.1.0:bafybeihurloujnbugvvrv5xegyrdnsgl6z6at5xilq7f5ynjdekijapt6q"
+.apy_estimation: &apy_estimation
+  <<: *aea_version
+  AEA_AGENT: "valory/apy_estimation:0.1.0:bafybeichfbqecgd62anequz3wonqfchrsluj473vrqqjsq2gqpwqo4vo5a"
```

Comment:
This hash should match that from packages.
As discussed on the call I have not made a test or anything, but I have also pre-pared this pr which automatically reads in these hashes. What do you think?

Reply:
https://github.com/valory-xyz/open-autonomy/pull/1273